crypto/fipsmodule/aes/asm/aesni-xts-avx512.pl

```diff
@@ -11,9 +11,11 @@
 # (https://github.com/intel/isa-l_crypto).
 #
 ######################################################################
-# The main building block of the loop is code that encrypts/decrypts 16 blocks
-# of data stitching with generation of tweak for the next 16 blocks, utilizing
+# The main building block of the loop is code that encrypts/decrypts 8/16 blocks
```

Comment:
Is it the input length that determines whether the algorithm will handle 8 or 16 blocks in parallel? I suggest adding to the comment how either one is chosen.

Reply:
Yes, main loop is selected based on input length. if the input data is greater than or equal to 256 bytes(16 blocks) then it would handle 16 blocks in parallel. If its between 128-256 bytes then 8 blocks would be handled in parallel.